tests/src/test/scala/system/health/BasicHealthTest.scala

```diff
@@ -109,7 +120,7 @@ class BasicHealthTest
                     }
 
                 println("Polling for activations")
-                val activations = wsk.activation.pollFor(N = 1, Some(triggerName), retries = 30)
+                val activations = wsk.activation.pollFor(N = 5, Some(triggerName), since = Some(start), retries = 30)
```

Comment:
It is important to note that we could make it a ridiculously large number and the test would not be affected. We are really only interested in 1 activation, and so this polling will almost always timeout after 30 seconds trying to obtain whatever number of activations we set here. The important thing is that the number is greater than 1 to handle what should be a very rare scenario.